tests/data/etlMapping_failed.yaml

```diff
@@ -0,0 +1,78 @@
+mappings:
```

Comment:
add an index without `doc_type` and make sure an error is returned and the validation fails